google-assistant-webserver/hassio_gassistant.py

```diff
@@ -32,7 +32,15 @@ def get(self):
         display_text = assistant.assist(text_query=text_query)
         return {'status': 'OK'}
 
+class StandardMessage(Resource):
+    def get(self):
+        message = request.args.get('message', default = 'This is a test!')
+        text_query = message
```

Comment:
Just a nit, but it might be nice to get rid of the unnecessary `text_query` variable altogether here and just use the `message` variable directly.